paasta_tools/contrib/service_shard_update.py

```diff
@@ -194,14 +240,28 @@ def main(args):
 
                     instance_config = {
                         "deploy_group": f"{deploy_prefix}.{args.shard_name}",
-                        "min_instances": args.min_instance_count,
-                        "max_instances": args.prod_max_instance_count
-                        if deploy_prefix == "prod"
-                        else args.non_prod_max_instance_count,
```

Comment:
@nemacysts (from #3829) how would you suggest testing paasta's behavior with no max/min instances?

Reply:
@karmeleon sorry, yesterday was a busy day!

i don't think we've setup the local paasta playground for autoscaling, but we should be able to create an instance in infrastage (or any test cluster) and see what happens

lemme do that real fast, actually :)

Reply:
whoops - replied in the wrong place: https://github.com/Yelp/paasta/pull/3829#discussion_r1569091874

Reply:
PAASTA-18283

Reply:
thanks for checking! to clarify, we're totally okay with min/max instances both being at 1 for brand new shards, since they typically see traffic ramp up gradually enough for autotune to deal with it. do you think it's safe to ship this branch now, or should we wait for that ticket to be completed?

Reply:
i'm trying to verify that autotune will do the "right" thing and setup min/max instances - but when i ran the rightsizer scripts locally it didn't seem to...that said, i'm not sure if it's cause i need to wait for splunk to have more data

Reply:
whoops, sorry - forgot about this: i think i hallucinated that autotune would fixup the autoscaling config: my test instance has not had autotune add min/max instances correctly :(

Reply:
(i.e., we'll either want to revert the {min,max}_instances changes here or tackle PAASTA-18283)